scripts/csv_to_geojson.py

```diff
@@ -0,0 +1,56 @@
+"""
+Simply script to convert a csv file of prediction results into a geojson
+
+Specify the CSV filepath and your confidence threshold. The CSV file should 
+contain tile indices in TMS coordinates. The confidence threshold is useful if 
+you ran prediction over a large area -- it will keep the resulting geojson 
+much smaller on disk. Set the threshold to 0.0 to include all tile predictions 
+in the geojson output.
+
+python csv_to_geojson.py my_results_file.csv 0.75
+
+Requires pygeotile, geojson, and mercantile
+"""
+
+import sys
+import csv
+import json
+
+from mercantile import feature, Tile
+from geojson import Feature
+from pygeotile.tile import Tile as Pygeo_tile
+
+with open(sys.argv[1], 'r') as csvfile:
+    with open('results.geojson', 'w') as results:
```

Comment:
can this file name be an argument (or pipe to process.stdout)?

Reply:
Yea, argparse the way to go here? I like click but maybe it's good to avoid an extra dependency?